build_tools/buildkite/cmake/android/arm64-v8a/benchmark2.yml

```diff
@@ -19,6 +20,7 @@ steps:
       IREE_DOCKER_WORKDIR: "/usr/src/github/iree"
     artifact_paths:
       - "benchmark-suites-${BUILDKITE_BUILD_NUMBER}.tgz"
+      - "source-mlir-models-${BUILDKITE_BUILD_NUMBER}.tgz"
       - "iree-android-tools-${BUILDKITE_BUILD_NUMBER}.tgz"
```

Comment:
![image](https://user-images.githubusercontent.com/4010439/163067592-fb515d01-0154-48db-89c1-dc6ddd417d6e.png)

280MB for the imported .mlir files, alongside the existing 1.14GB compiled .vmfb files and 8Mb binary tools (at this commit - those sizes will no doubt increase over time :P)

If we're okay with the storage increase, I'd find this useful. The Colab notebook approach I was taking on https://github.com/google/iree/pull/8846 is flexible and won't use extra cloud storage, but this is easier to use (works across all commits, no need to fiddle with Colab or installs tools yourself)

Reply:
Yes let's have this for now to make life easier. We can always rethink if it becomes a problem later. :)